tests/test_image_stack_py.py

```diff
@@ -1,6 +1,7 @@
 import logging
 import numpy as np
 import unittest
+import pytest
```

Comment:
Pytest isn't included as part of KBMOD. You can use: 

```
self.assertRaises(ValueError, function_name, params)
```

instead. See test_clustering_filters.py for an example